src/components/search/GlobalSearchField.js

```diff
@@ -274,7 +275,7 @@ function AppsSearchOption(props) {
             <SearchOption
                 primary={selectedOption.name}
                 secondary={selectedOption.description}
-                icon={<img src="/icon-apps.png" alt={t("apps")} />}
```

Comment:
GitHub Actions is reporting a warning above that `const { t } = useTranslation("common");` is no longer used.